### DISCUSSION TITLE
Welcome to tsai Discussions!

### DISCUSSION THREAD

**Discussion by oguiza** [timeseriesAI/tsai#32] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vrodriguezf** [comment]
Great idea Ignacio! My name is Víctor, I work as an assistant professor at the department of computer systems engineering of Universidad Politécnica de Madrid, Spain. I've been working with time series in one way or another since my PhD, and since I did the fastai course, I'm super interested in using deep learning for time series as part of the research projects I have here. 

Unfortunately, my time for coding is very limited :( but I follow the trends and news in time series research, so I can provide insights and papers that can help to guide people in the community.

Take care!

**Comment by kiview** [comment]
Hi and thanks for creating this great project!

I am an open-source developer and went back to academia 2 years ago to do my Ph.D. (in computer science / distributed systems engineering and about distributed ledger technology). Through my involvement with academia and research, I started to collaborate together with my wife @neslihanedes on different research projects and support her on the computing part of her behavior psychology research (regarding birds).

We are currently working on the behavior classification of birds, based on tracked 2D coordinate data. I thought this is something that should be solvable using supervised machine learning and I thought this could be formulated in abstract terms as anomaly detection and classification in time series data. As someone without any background in AI or machine learning, I started to dive into this field and found many useful papers, but it was hard to implement the concepts without a good understanding of the ecosystem and programming DSLs. After getting good results with a hand-rolled 1D CNN architecture in Tensorflow I doubled down on Googling and finally found *tsai* when looking into [fastai](https://github.com/fastai/fastai). 

At a first glance, tsai provides API for nearly everything we want to try out and it seems like the perfect abstraction so we don't have to focus too much on the implementation details. Many operations also seem much faster than my own non-idiomatic numpy interactions. The first tests are very promising and we are quite optimistic. 

We hope we will be able to finish a paper soon and we will happily reference tsai 🙂.

I came here to ask some usage questions, which I will put in its own posts, but wanted to share my background and excitement about tsai first.

**Reply by oguiza** [reply]
HI @kiview, thanks for providing your background. 
It seems to be a very interesting project. As you say, DL is usually good at this type of task. There are many different approaches you'll be able to try with `tsai`. 
And thanks for considering referencing `tsai` if it ends up being useful in your work. I'll try to answer your questions as you post them.

**Comment by masonmahaffey** [comment]
This is such an incredible project :)

Just wanted to stop by and say I'm super impressed by the work you guys have accomplished and hope to maybe contribute in the future. Keep up the amazing work 👍

**Comment by oguiza** [comment]
Wow! Thank you! I really appreciate your comment. 
I hope it is useful in your time series problems.
